web-frontend/src/pages/TransferCoins/SendCoins/Send.tsx

```diff
@@ -0,0 +1,176 @@
+import { Balance, Button, Input } from '@massalabs/react-ui-kit';
+import { FiUsers } from 'react-icons/fi';
+import useResource from '../../../custom/api/useResource';
+import { AccountObject } from '../../../models/AccountModel';
+import { useLocation } from 'react-router-dom';
+import { parseForm } from '../../../utils/parseForm';
+import { useState } from 'react';
+
+// remove padding from Balance component
+
+interface IErrorObject {
+  amount?: string;
+  fees?: string;
+  recipient?: string;
+  error?: string;
+}
+
+function Send() {
+  const { state } = useLocation();
+  const nickname = state.nickname;
+  const { data: accounts = [] } = useResource<AccountObject[]>('accounts');
+  const account = accounts.find((account) => account.nickname === nickname);
+  const balance = parseInt(account!.candidateBalance);
+  const [amount, setAmount] = useState<number>(0);
+  const [error, setError] = useState<IErrorObject | null>(null);
+  const [recipient, setRecipient] = useState('');
+  const selectPercentClass = 'mr-3.5 hover:cursor-pointer';
+
+  // refactor to switch => todo
+  // Check to see if address exists in our system
+
+  function validate(e: any) {
+    const formObject = parseForm(e);
+    const { amount, recipient } = formObject;
+
+    if (!amount) {
+      setError({ amount: 'Amount is required' });
```

Comment:
Any reason to no use I18n here and in the whole page BTW!? 🤔

Reply:
would you like it to be done in this PR or a separate ? :)

Reply:
done